core/adminserver/admin_server_test.go

```diff
@@ -64,14 +64,8 @@ func newMiniEnv(ctx context.Context, t *testing.T) (*miniEnv, error) {
 	if err != nil {
 		return nil, fmt.Errorf("error starting fake server: %v", err)
 	}
-	srv := &Server{
```

Comment:
Be careful here, I don't think I'm actually setting all the variables in server - there's a few I don't need for this test. 

`New` can work, but you'll need to pass `nil` for a few items.